test/numbers.jl

```diff
@@ -2137,7 +2137,8 @@ rationalize(nextfloat(0.0)) == 0//1
 # rational-exponent promotion rules (issue #3155):
 @test 2.0f0^(1//3) == 2.0f0^(1.0f0/3)
 @test 2^(1//3) == 2^(1/3)
-
+# no loss of precision for rational powers (issue #18114)
+@test_approx_eq_eps BigFloat(2)^(BigFloat(1)/BigFloat(3)) BigFloat(2)^(1//3) 1e-20
```

Comment:
Cleaner to do `@test BigFloat(2)^(BigFloat(1)/BigFloat(3)) ≈ BigFloat(2)^(1//3)` ... this will test to sqrt(eps) precision by default, about 4e-39.